src/main/java/module-info.java

```diff
@@ -181,5 +181,6 @@
     requires mslinks;
     requires org.antlr.antlr4.runtime;
     requires org.libreoffice.uno;
+    requires jeuclid.core;
```

Comment:
Where is this used?

Reply:
Also take ` (alphabetically)` serious. `j` is after `dd`, but not after `org`

Reply:
Sorry, that wrong IntelliJ suggestion and it imported it